tests/helpers.py

```diff
@@ -2330,3 +2330,24 @@ def retrieve_default_ingress_crt():
 
     with open(constants.DEFAULT_INGRESS_CRT_LOCAL_PATH, 'w') as crtfile:
         crtfile.write(decoded_crt)
+
+
+def get_pv_size(storageclass=None):
```

Comment:
please place this function in a file under ocs_ci/ 
maybe under ocs_ci/ocs/resources/pvc.py

Also, I see this function returns a list with the sizes of all the storageclass PVs.
What's the use case of such functionality? How about changing this function to return the size of a single PV and you'll create a list of the PVs outside. This will make this function more generic and usable

Reply:
Here I'm expecting to return PV size so that I can make use of PV size in storagecluster cr while creating it for BM env. As per doc, we are requesting customers to provide PV size as requested storage size that will help to avoid picking up lower size PV for creating OSD.

Reply:
in some cases might happen that all pv size are different so while requestion storage size in storage cluster cr we need to provide lower pv size volume then only all osd will come up if we provide big pv size small size pv will not be included and that might lead osd stuck in pending case